storybook/CustomPropTable.js

```diff
@@ -0,0 +1,123 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import PropVal from '@storybook/addon-info/dist/components/PropVal'
+import PrettyPropType from '@storybook/addon-info/dist/components/types/PrettyPropType'
+
+const Table = props => <table {...props} />
```

Comment:
I adopted coding style of @storybook/addon-info.

I like that all table elements are declared side by side.

It would be possible to use directly.